source/code/util/xml/XMLReader.cpp

```diff
@@ -1935,18 +1935,16 @@ void XMLReader::XML_PutError( void )
 */
 void XMLReader::XML_Raise(const char* format, ...)
 {
-    int n;
     va_list ap;
     memset(&ap, 0, sizeof(ap));
 
     m_Status = -1;
     m_Message[0] = u8_NullChar;
 
     va_start(ap, format);
-    n = vsnprintf(m_Message, sizeof(m_Message), format, ap);
+    vsnprintf(m_Message, sizeof(m_Message), format, ap);
```

Comment:
Previous code didn't do that, and I just made it compile.

This routine, while it can return an error, generally just returns the number of characters, and clearly this code didn't care what that was ...